UnityProject/Assets/Scenes/AsteroidScenes/Asteroid05.unity

```diff
@@ -1,6650 +0,0 @@
-%YAML 1.1
-%TAG !u! tag:unity3d.com,2011:
---- !u!29 &1
-OcclusionCullingSettings:
-  m_ObjectHideFlags: 0
-  serializedVersion: 2
-  m_OcclusionBakeSettings:
-    smallestOccluder: 5
```

Comment:
why deleting?

Reply:
Well, I accidentally added an additional 21 unintended changes

Reply:
And I'm not exactly sure how to revert those change